include/Yash.h

```diff
@@ -299,10 +449,13 @@ class Yash {
     static constexpr const char* s_clearLine = "\033[2K\033[100D";
     static constexpr const char* s_clearScreen = "\033[2J\x1B[H";
     static constexpr const char* s_clearCharacter = "\033[1D \033[1D";
+    static constexpr const char *s_moveCursorForward = "\033[1C";
+    static constexpr const char *s_moveCursorBackward = "\033[1D";
     static constexpr const char* s_commandDelimiter = " ";
     std::map<std::string, Function> m_functions;
     std::vector<std::string> m_commands;
     std::vector<std::string>::const_iterator m_commandsIndex;
+    size_t pos{0};
```

Comment:
Maybe m_position would be better? Think we also miss clang format for the missing space

Reply:
should be fixed